native/wasmex/src/wit.rs

```diff
@@ -0,0 +1,18 @@
+use rustler::{NifResult, Term};
+use wit_parser::{Resolve, WorldItem};
+
+#[rustler::nif(name = "wit_exported_functions")]
+pub fn exported_functions(env: rustler::Env, path: String, wit: String) -> NifResult<Term> {
+    let mut resolve = Resolve::new();
+    let id = resolve.push_str(path, &wit).unwrap();
+    let world_id = resolve.select_world(id, None).unwrap();
+    let exports = &resolve.worlds[world_id].exports;
+    let exported_functions = exports
+        .iter()
+        .filter_map(|(_key, value)| match value {
+            WorldItem::Function(function) => Some((&function.name, function.params.len())),
```

Comment:
would be nifty if we could also return the types (params types and return types) for nicer introspection/debugging/tooling later

Reply:
also to prepare for elixir types (whenever they become available) - would be aze if wasmex could created typed interfaces/function definitions right away

Reply:
Yep, i did this way for expediency in getting something working we need a better API to get back data from wit parsing.